lib/pushmi_pullyu/cli.rb

```diff
@@ -89,6 +95,20 @@ def parse_options(argv)
         config.monitor = true
       end
 
+      opts.on('-rh', '--redis-host IP', 'Host IP of Redis instance to read from.'\
```

Comment:
Yeah exactly I'll have a default location with a default `config/pushmi_pullyu.yml`. If you ever need to change that then you can provide your own via command line. 

Aka `pushmi_pullyu -C config/production.yml`